modules/ROOT/pages/design-create-publish-api-specs.adoc

```diff
@@ -5,11 +5,15 @@ endif::[]
 :imagesdir: ../assets/images
 :page-aliases: general:getting-started:design-an-api.adoc, design-migrating-ms.adoc
 
```

Comment:
I suggest making the titles in this PR more robust: What _about_ API Designer?
If this is truly an overview, then you may use that term (although I find it pretty vague); if you can define the focus of the topic, that is better:
= Design <what?> Using the API Designer Web-Based Interface?

Reply:
Later...now that I see the final subtopic in this set, I think that it should be adapted to be the topic introduction, and that the other subtopics be converted into a bullet list of what "you" can do with API Designer.